test/utils/addChaiAssertions.ts

```diff
@@ -7,7 +7,7 @@ declare global {
       /**
        * Matcher with useful error messages if the dates don't match.
        */
-      toEqualDateTime(expected: Date): void;
+      toEqualDateTime(expected: any): void;
```

Comment:
This line was already very wrong, the actual value was `TDate` and now it's `TDate | string`